scripts/symlink_git_hooks.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+base=$HOME
+
+# If no HOME directory was set, use a BASE directory that is /home/dev
+if [ ! $base ]
+then
+    base=/home/dev
+fi
+
+source_directory=$base/Sources/OSGP
+target_directory=*/.git/hooks
+
+cd $source_directory
+git_hook_directories=$(ls -d $TARGETDIR)
```

Comment:
Does this script work? I would expect `$target_directory` instead of `$TARGETDIR` based on line 12, and the fact that TARGET_DIR does not appear to be defined anywhere in this script.